addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -5828,6 +5828,9 @@ public Action:OnTakeDamage(client, &attacker, &inflictor, &Float:damage, &damage
 		new boss=GetBossIndex(client);
 		if(boss!=-1)
 		{
+			new String:bossName[768];
```

Comment:
64 is the limit everywhere else in the code so it'd be pretty pointless to have just one spot with 768.